packages/search/src/use-fuzzy-search.ts

```diff
@@ -4,6 +4,7 @@ import { useMemo } from 'react';
 const defaultOptions = {
 	threshold: 0.4,
 	distance: 20,
```

Comment:
@danielbachhuber I was reviewing https://github.com/Automattic/wp-calypso/issues/66248, and I found out that when we use `ignoreLocation`, it ignores the `distance`. In this case, it makes sense to remove `distance` to avoid confusion.

Reply:
@wojtekn Sounds good! Created #66342